Fabric.Identity.API/Startup.cs

```diff
@@ -201,6 +202,18 @@ public void ConfigureServices(IServiceCollection services)
             }
 
             app.UseExceptionHandler("/Home/UnauthorizedError");
+            app.UseStatusCodePages(
```

Comment:
Is it better to use UseStatusCodePagesWithRedirects? It seems like the status code might change to a 302 with that though.  What happens to the status code in this case?

Reply:
I looked at that as well. Using UseStatusCodePagesWithRedirects never shows you had a 401 in the dev tools. It just redirects like nothing was wrong. The better one was UseStatusCodePagesWithReExecute but that wouldn't work. It caused more issues and sometimes when clicking the Windows button to login, instead of seeing the login, it would just throw a 500 error. Using UseStatusCodePages is similar to the ReExecute just more code. You still see the 401 status code in the dev tools with UseStatusCodePages.